docs/development/windows_support.md

```diff
@@ -180,6 +180,14 @@ If you prefer to install tools manually, you will need:
 > - You can also tell CMake to use MSVC's tools explicitly with
 >   `-DCMAKE_C_COMPILER=cl.exe -DCMAKE_CXX_COMPILER=cl.exe -DCMAKE_LINKER=link.exe`
 
+### Locale
+
+If the build system is a non-English system. Make sure to switch to ``utf-8``.
+
+```
+chcp 65001
+```
```

Comment:
nit: you could set a language hint on this fenced code block: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting

Other code blocks here use `bash`, but that doesn't color this particular text:

no language
```
chcp 65001
```

bash
```bash
chcp 65001
```

powershell
```powershell
chcp 65001
```

cmd
```cmd
chcp 65001
```